lualib/resty/events/broker.lua

```diff
@@ -62,12 +69,16 @@ function _M.run()
 
     local queue = que.new()
 
-    _clients[conn] = queue
+    self._clients[conn] = queue
 
     local read_thread = spawn(function()
         while not exiting() do
             local data, err = conn:recv_frame()
```

Comment:
The `DEFAULT_TIMEOUT` seems to be `5` secs, should we be able to control it somehow? E.g. here we set `kong` default for clustering timeout: https://github.com/Kong/kong/blob/5b7676cee6a1f16d5575db78eda16410047a4ac8/kong/constants.lua#L200

Because this is connection to same node, perhaps the timeout cal be smaller by default, like `1 secs`, it will help with faster `graceful shutdown`, and perhaps quicker `reload` too.

Reply:
done.